War Engines first pass (#28)
GameBrains/er-core: modules/special-rules/pages/war-engine-catastrophic-damage.adoc

= War engine catastrophic damage

> A destroyed war engine may be only wrecked or killed -- out of the battle, but possibly salvageable.
> Or, it may be completely annihilated in a spectacular fashion that presents a hazard to nearby units, whether friend or foe.

Reviewer: Looks good. I just have some tweaks for a variety of minor reasons grammatical or semantic reasons. I was going to suggest this:

```suggestion
A destroyed war engine or bio-construct might be only wrecked or killed -- out of the battle, but potentially salvageable.
Or, it may be annihilated in a spectacular fashion that presents a hazard to nearby units, whether friend or foe.
```
...but it occurs to me that you can't really 'salvage' a 'killed' bio-construct...! A little more thought required I think.

Author: Yeah, I'm not that happy with 'salvage' either.

Reviewer: A couple of ideas for this. I favour the first one, but I could live with the second one if you feel opposed to it.

### (1) Consolidate on the term 'wrecked'

We could simply use the term 'wrecked' in all cases. I don't see why this shouldn't apply to Tyranid bio-constructs too. On their actual data sheets, we can still cast the fluff in terms of 'death' — and this need not conflict with the result of 'Wrecked', in my opinion:

>**1–2 Terminal Bleeding — Wrecked**: The Dominatrix gushes ichor and bodily fluids as it dies in a sodden heap. 

Seems fine to me. And after all, it's no less odd to speak of 'repairs' for bio-constructs — but we don't let that worry us.

I think that this is the most straightforward solution. We could also drop the 'salvageable' bit from the text here, thus:

```suggestion
A destroyed war engine may be only wrecked.
Or, it may be completely annihilated in a spectacular fashion that presents a hazard to nearby units, whether friend or foe.
```

---

### (2) Just drop the reference to 'salvageable'

```suggestion
A destroyed war engine might be only wrecked (or killed, in the case of the giant Tyranid bio-constructs).
Or, it may be annihilated in a spectacular fashion that presents a hazard to nearby units, whether friend or foe.
```
This seems okay, it's just a bit redundant to have both 'wrecked' and 'killed' as synonyms where the rules are concerned (and of course 'killed' can be rather more general like 'destroyed' can, although I've been careful to avoid that usage).

Not a big problem because, as far as I can recall, we don't really need to refer to the 'Wrecked' or 'Killed' states outside of the data sheets themselves. So we shouldn't need untidy constructions like 'Wrecked or Killed'. The real problem was the multiple different meanings for 'destroyed' when we do actually refer specifically to the 'Destroyed' outcome in the army morale rules — which we solve by use of 'Annihilated'.

Reviewer: I think I'll go ahead and commit my suggestion (1). After all, special variation of 'Wrecked' terminology in favour of 'Killed' seems doubly needless when we'll still call the Tyranid bio-constructs 'war engines' despite the fact that they aren't really mechanical 'engines' like the rest...

---

When a War Engine unit's current Damage Capacity value reaches `0`, it suffers *catastrophic damage*.
Roll a D6 and check the result against the Catastrophic Damage Table on that unit's data sheet.

> {blank}[TODO: Ideally, we should change some terminology to disambiguate the general usage of the term 'destroyed' from the specific result of 'Destroyed' (rather than 'Wrecked') on catastrophic damage tables.]
> // IJW reply: Biotitans are 'Killed' rather than 'Wrecked'. I suggest 'Annihilated' to replace 'Destroyed' in the results.

Reviewer: Yes, I agree — 'Annihilated' disambiguates well so that we have no overlap with the more usual use of 'destroyed'.
```suggestion
```
